linkerd/tls/src/main/scala/io/buoyant/linkerd/tls/TlsUtils.scala

```diff
@@ -128,6 +148,12 @@ object TlsUtils {
     |URI.1 = https://buoyant.io
     |""".stripMargin
 
+  def dnsAltNames(names: Seq[String]): String = {
```

Comment:
NIT: I would put this function right under `addDnsAltNamesInConfig` so that all this directly-related code is together.